tests/test_tavily_search.py

```
@@ -0,0 +1,47 @@
+        for idx, result in enumerate(results, 1):
+            print(f"\n--- Result {idx} ---")
+            print(f"Title: {result.title}")
+            print(f"URL: {result.url}")
+            print(f"Site: {result.site_name}")
+            if result.snippet:
+                snippet = result.snippet[:100] + "..." if len(result.snippet) > 100 else result.snippet
+                print(f"Snippet: {snippet}")
+            print(f"Score: {result.score}")
+            
+        assert len(results) > 0, "No search results returned"
+        return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)